### PR TITLE
feature: dm-315 notification emails

### PR DESCRIPTION
**Pull request by itsmostafa**
Add ses sending email functionality
Add website is live email notification on site launch

## 💭 Description
An email will now be sent confirming a website is launched once it is live.

## ✅ Checklist

- [X] My code follows the code style of this project.
- [X] My changes have been adequately tested locally.
- [X] All automated tests are passing.
- [X] I have updated/added required automated tests.
- [X] Pre-commit checks are passing.

### CODE REVIEW

File: src/utils/aws/ses.py [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+from settings import APP_ENV, APP_NAME, SES_ASSUME_ROLE_ARN, SMTP_FROM
+from utils.aws import sts
 
 route53 = boto3.client("route53")
-ses = boto3.client("ses")
+
+if SES_ASSUME_ROLE_ARN:
+    ses = sts.assume_role_client("ses", SES_ASSUME_ROLE_ARN)
+else:
+    ses = boto3.client("ses")
```
Comment (zenine07) [inline src/utils/aws/ses.py]: This is not going to work having it like this. We need to assume the SES role when sending email, but we would not want to use that assumed role when creating an email address for one of our domains.

File: src/utils/notifications.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    def send(self):
+        """Send Email."""
+        # Get user email
+        dm_user = user_manager.get(filter_data={"Username": g.username})
+        send_to = "".join(
+            attribute["Value"]
+            for attribute in dm_user["Attributes"]
+            if attribute["Name"] == "email"
+        )
```
Comment (zenine07) [inline src/utils/notifications.py]: May not always be sent to only the dm_user. I would maybe provide options as far as sending. Provide options for all admins, all application members, all users, and only authenticated user. Then I would just do a bcc to all members included in the list.

**Review by zenine07 — CHANGES_REQUESTED** [review]
I think once those two things are settled, we'll have a great notification service with all the flexibility we need. Great job so far, loving how simple it is, yet very extensible.

File: src/utils/notifications.py (hunk 2 above)
Comment (itsmostafa) [reply src/utils/notifications.py]: Let's discuss this further sometime today.

**Review by dylanj1752 — APPROVED** [review]
LGTM

**Pull request merged**